src/index.js

```diff
@@ -10,6 +10,8 @@ if (!Promise) {
   Promise.finally = require('promise-polyfill').finally
 }
 
+const abort = () => { throw new Error('Aborted') }
```

Comment:
In order to mimic the original behaviour, I'd suggest to throw the following error : 

```suggestion
const abort = () => { throw new DOMException('The operation was aborted', 'AbortError'); }
```
In https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort : 
> Note: When abort() is called, the fetch() promise rejects with an AbortError.

And in https://developer.mozilla.org/fr/docs/Web/API/DOMException : 
> AbortError
    The operation was aborted. (Legacy code value: 20 and legacy constant name: ABORT_ERR)

(Can't wait to see this PR merged)